tools/scope_upload_classification.py

```diff
@@ -58,12 +72,17 @@ def upload_classification(
                     cls_list += [cls]
                     probs[cls] = row.iloc[-1 * len(classification) + i]
 
-        ra, dec, period = float(row.ra), float(row.dec), float(row.period)
+        ra, dec = float(row.ra), float(row.dec)
+
+        # Check for and assign period
+        period = None  # default
+        if 'period' in columns:
+            period = float(row.period)
+            if math.isnan(period):
```

Comment:
Maybe throw a warning if a period is NaN?

Reply:
Makes sense, I added separate warnings for NaN and missing period column.